internal/compiler-bridge/src/main/scala/xsbt/STJ.scala

```diff
@@ -0,0 +1,79 @@
+package xsbt
+import java.io.File
+import java.nio.file.Paths
+import java.util.zip.ZipFile
+
+class STJ(outputDirs: Iterable[File]) {
```

Comment:
Could you add Scaladoc explaining what STJ stand for, and what this class does before we forget?

Reply:
Should this class be marked final?